artman/tasks/gapic_tasks.py

```diff
@@ -103,7 +103,7 @@ class GapicCodeGenTask(task_base.TaskBase):
     def execute(self, language, toolkit_path, descriptor_set, service_yaml,
```

Comment:
A function with 14 parameters makes me cry. What can go wrong here :)

(obviously needs a refactor, but I'm guessing it will never happen)